core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-favorite-page/dot-favorite-page.component.html

```diff
@@ -7,15 +7,38 @@
 >
     <form class="p-fluid" *ngIf="form" [formGroup]="form" data-testId="form">
```

Comment:
<img width="1128" alt="image" src="https://user-images.githubusercontent.com/751424/211675184-ecfae04d-20eb-41f0-bdfa-ab9c724b84ff.png">

Form is out of bounds.

Reply:
fixed!